README.md

```diff
@@ -17,16 +17,21 @@ MINI2P_toolbox includes the codes, softwares, 3D models, protocols, and etc. for
 
 2) [Protocols](Protocols)
 
-    a) [P1–Shopping and Machining list](https://github.com/kavli-ntnu/MINI2P_toolbox/blob/main/Protocols/P1%20-%20Shopping%20%20Machining%20List%20.pdf). This document lists each essential component with the supplier, the product name, the model (or item reference) and its approximate price in Euro. 2D Drawings and 3D models of most components are available [here](https://github.com/kavli-ntnu/MINI2P_toolbox/tree/main/Hardware) .
+    a) [P1–Shopping and Machining list](https://github.com/kavli-ntnu/MINI2P_toolbox/blob/f123bb4d4f2f6fd875c3539214261601fda34116/Protocols/P1%20-%20Shopping%20%20Machining%20List%20.pdf). This document lists each essential component with the supplier, the product name, the model (or item reference) and its approximate price in Euro. 2D Drawings and 3D models of most components are available [here](https://github.com/kavli-ntnu/MINI2P_toolbox/tree/main/Hardware) .
```

Comment:
This creates a link to a specific version of the shopping/machining list, and will keep pointing to that old version even after you update the list in future unless the link is also updated.